src/throttler-core.module.ts

```diff
@@ -0,0 +1,24 @@
+import { createConfigurableDynamicRootModule } from '@golevelup/nestjs-modules';
+import { Global, Module } from '@nestjs/common';
+import { ThrottlerStorage } from './throttler-storage.interface';
+import { THROTTLER_OPTIONS } from './throttler.constants';
+import { ThrottlerOptions } from './throttler.interface';
+import { ThrottlerStorageService } from './throttler.service';
+
+@Global()
+@Module({})
+export class ThrottlerCoreModule extends createConfigurableDynamicRootModule<
```

Comment:
Following other official modules, could we inline this logic (`createConfigurableDynamicRootModule`) instead of using another, external dependency? You can see examples in other `@nestjs/...` packages

Reply:
Yeah, we can use the standard dynamic module setup. Kinda forgot about that when creating the module and I've gotten used to using WonderPanda's `createConfigurableDynamicRootModule` cause it makes things so easy. No problem though :smile: